src/main/java/com/minecolonies/coremod/colony/buildings/BuildingHome.java

```diff
@@ -199,11 +199,6 @@ public void removeCitizen(@NotNull final CitizenData citizen)
     @Override
     public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
     {
-        if (event.phase != TickEvent.Phase.END)
```

Comment:
Why is this removed?

Reply:
because the building get called over the colonyManager don't they? So we don't have to check it twice.

Reply:
Put it in just to be sure. When we rewrite we will face this anyway. So we can do a double check then.